content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -25,81 +31,73 @@ depends on the stage of the manufacturing process.
+pop-up window.
 
-Components can also be scrapped from the manufacturing tablet view. To do so, select the
-:guilabel:`Work Orders` tab on a manufacturing order, then click the :guilabel:`📱 (tablet view)`
-icon for a work order.
+Scrap component from Shop Floor
+-------------------------------
 
-.. image:: scrap_manufacturing/tablet-view-icon.png
-   :align: center
-   :alt: The tablet view icon for a work order.
+To scrap a component from the *Shop Floor* module, begin by navigating to :menuselection:`Shop
+Floor`. Then, either click the :guilabel:`⋮ (three vertical dots)` button on an |MO| card, or select
+a work center from the top navigation, and click the :guilabel:`⋮ (three vertical dots)` button on a
+work order card.
 
-With tablet view open, click the :guilabel:`☰ (menu)` button at the top left of the screen, then
-select the :guilabel:`Scrap` button on the :guilabel:`Menu` pop-up window. The :guilabel:`Scrap`
-pop-up window then appears.
+Either method opens the :guilabel:`What do you want to do?` pop-up window. Click the
+:guilabel:`Scrap` button on the window to open the :guilabel:`Scrap` pop-up window.
 
-.. image:: scrap_manufacturing/tablet-scrap-button.png
-   :align: center
-   :alt: The Scrap button on the Menu pop-up window of the manufacturing tablet view.
+Scrap pop-up window
+===================
 
-Finally, select a component from the :guilabel:`Product` drop-down menu and enter the quantity being
-scrapped in the :guilabel:`Quantity` field. Click :guilabel:`Done` to scrap the component.
+After opening the scrap pop-up window using one of the methods :ref:`detailed above
+<scrap_components/scrap-window>`, select the component or finished product being scrapped, from the
+:guilabel:`Product` drop-down menu. In the :guilabel:`Quantity` field, enter the quantity being
+scrapped.
 
-Scrap finished products
-=======================
+By default, the :guilabel:`Source Location` field is set to the warehouse's pre-production location,
+while the :guilabel:`Scrap Location` field is set to the :guilabel:`Virtual Locations/Scrap`
+location. If either the source or scrap location should be changed, select a different location from
+their respective drop-down menus.
 
-Odoo also allows for finished products to be scrapped from a manufacturing order once the order is
-completed. After clicking :guilabel:`Mark as Done`, click the :guilabel:`Scrap` button to make the
-:guilabel:`Scrap` pop-up window appear.
+Enable the :guilabel:`Replenish Scrapped Quantities` checkbox if a picking order should be created
+to replace the scrapped component(s) upon confirmation of the scrap order. This option should only
+be enabled for warehouses with :doc:`two-step <two_step_manufacturing>` or :doc:`three-step
+<three_step_manufacturing>` manufacturing enabled, since components are not picked as part of the
+:doc:`one-step <one_step_manufacturing>` manufacturing process.
```

Comment:
```suggestion
Enable the :guilabel:`Replenish Scrapped Quantities` checkbox if a picking order should be created
to replace the scrapped component(s) upon confirmation of the scrap order.

This option should only be enabled for warehouses with :doc:`two-step <two_step_manufacturing>` or
:doc:`three-step <three_step_manufacturing>` manufacturing enabled, since components are not picked
as part of the :doc:`one-step <one_step_manufacturing>` manufacturing process.
```

Reply:
I'd like to keep this paragraph together since there are only two sentences, and the second is directly referencing the first.